### PR TITLE
feat: ab setup added to filters page for new challenge header

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- ab setup added to filters page for new challenge header

### CODE REVIEW

File: src/pages/Lend/LoanSearchPage.vue [hunk 1]
```diff
@@ -137,6 +139,15 @@ export default {
 				'EXP-CORE-1205-May2023'
 			);
 		}
+
+		const challengeHeaderExpData = trackExperimentVersion(
+			this.apollo,
+			this.$kvTrackEvent,
+			'borrower-profile',
```
Comment (dyersituations) [inline src/pages/Lend/LoanSearchPage.vue:146]: `Lending`

File: src/pages/Lend/LoanSearchPage.vue [hunk 2]
```diff
@@ -137,6 +139,15 @@ export default {
...
+			CHALLENGE_HEADER_EXP,
+			'EXP-ACK-1038-MAR2024',
```
Comment (dyersituations) [inline src/pages/Lend/LoanSearchPage.vue:148]: Looks like the wrong key

File: src/pages/Lend/LoanSearchPage.vue [hunk 3]
```diff
@@ -137,6 +139,15 @@ export default {
...
+		);
+		this.enableChallengeHeader = challengeHeaderExpData.version === 'b';
```
Comment (dyersituations) [inline src/pages/Lend/LoanSearchPage.vue:150]: challengeHeaderExpData?.version

File: src/pages/Lend/LoanSearchPage.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/pages/Lend/LoanSearchPage.vue:148]: `ACK-1038` is the epic for this...

Comment (dyersituations) [reply src/pages/Lend/LoanSearchPage.vue:148]: Oh I was looking at the story, cool.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**